### ISSUE TITLE
If an item in a collection is marked as active the secondary content becomes invisible

### ISSUE THREAD

**Issue by zlepper** [Dogfalo/materialize#1927]
```
        <ul class="collection with-header">
              <li class="collection-header"><h4>First Names</h4></li>
              <li class="collection-item">Alvin<a href="#!" class="secondary-content"><i class="material-icons">send</i></a></li>
              <li class="collection-item">Alvin<a href="#!" class="secondary-content"><i class="material-icons">send</i></a></li>
              <li class="collection-item">Alvin<a href="#!" class="secondary-content"><i class="material-icons">send</i></a></li>
              <li class="collection-item active">Alvin<a href="#!" class="secondary-content"><i class="material-icons">send</i></a></li>
            </ul>
```

The last secondary-content in the last box is invisible.

**Comment by AEgan** [comment]
I tried this in codepen to check it out.
The "send" link in `secondary-content` is rendered for me, but the link's color is the same as the background-color of the `collection-item` li

I changed that to red here as an example
http://codepen.io/anon/pen/LVKBMQ

**Comment by acburst** [comment]
Fixed in 5df452c
